### ISSUE TITLE
Feature request: generating HGVS compliant nomenclature for a selected variant

### ISSUE THREAD

**Issue by dsonkin** [igvteam/igv#1555]
Hello,

It should be useful to be able to right-click on a variant in the IGV viewer and have an option to generate HGVS compliant nomenclature for a variant.

Thank you,
Dmitriy

**Comment by jrobinso** [comment]
Hi, I don't really know what that is.   What exactly are you requesting?

**Comment by dsonkin** [comment]
Hi,

HGVS Nomenclature is described at https://hgvs-nomenclature.org/stable. I've attached zip file with small BAM and corresponding index file (GRCh37/hg19), also attached is snapshot of IGV view for this BAM file opened using following genomic coordinates chr17:7,574,014-7,574,053. This BAM contains TP53 variant NC_000017.10:g.7574034C>T which is marked in red across reads in the middle of IGV view. If copy read details function is used while pointing mouse cursor to this variant, read details include Location = chr17:7,574,034 and Base = T. It would be very useful if variant HGVS genomic description and MANE Transcript HGVS description (if available) would be included in addition to location and base, in this case it would be NC_000017.10:g.7574034C>T and NM_000546.6:c.994-1G>A.      

[G30631.SU-DHL-10.1_TP53_min.zip](https://github.com/user-attachments/files/16875297/G30631.SU-DHL-10.1_TP53_min.zip)
![IGV_TP53_example](https://github.com/user-attachments/assets/c7f3d9eb-10f6-4167-b7b1-80f5dfb5b223)

Thank you,
Dmitriy

**Comment by jrobinso** [comment]
I don't think this is practical to do in IGV.    IGV does not call variants, and we don't want to claim to call variants.   Your particular example seems straightforward, but imagine multiple mismatches at that location(A and G in addition to T, as well as some indels).   What is the HGVS description of that?    

Also, there are likely several transcripts there, I think the gene track is probably collapsed.

**Comment by jrobinso** [comment]
Apologies for the long delay.  I'm looking into this more deeply now.  One problem with your example, the Mane transcripts are only created for GRCh38 / hg38.

**Comment by dsonkin** [comment]
Which stable version of IGV will include this functionality?
Thank you
